installer.c

```diff
@@ -594,7 +594,7 @@ static void installer_flash_cmd(INTN argc, CHAR8 **argv)
 	for (int i = 0; i <  num; i++)
 		numname[i] = NULL;
 
-	if (argc > 4) {
```

Comment:
Needs double check the original code.  seems there is no args==3 test case.

Reply:
args==3 in else for if (argc > 3)